lib/engine/g_18_zoo/company.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
```

Comment:
can we delete this? i think it's unnecessary, we can achieve phase some other way, i can help you figure that out once i understand what phase is